demo/App.tsx

```diff
@@ -6,10 +6,82 @@
  *
  */
 
+import * as Constants from './src/constants';
+import Hyperview from 'hyperview';
 import './src/gesture-handler';
+import React, { useState } from 'react';
+import { fetchWrapper, formatDate } from './src/helpers';
+import {
+  SafeAreaInsetsContext,
+  SafeAreaProvider,
+} from 'react-native-safe-area-context';
+import HyperviewScreen from './src/HyperviewScreen';
+import { View } from 'react-native';
 import Navigator from './src/Navigator';
-import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
 
-export default function App() {
-  return <Navigator />;
+const NAMESPACE_URI = 'https://instawork.com/hyperview-set-navigator-source';
+
+/**
+ * Define expected values
+ */
+enum NavigationSource {
+  HXML = 'hxml',
+  EXTERNAL = 'external',
 }
+
+export default () => {
+  const [useHXML, setUseXML] = useState<boolean>(false);
+
+  /**
+   * Create a custom behavior to toggle using HXML to define the navigator hierarchy
+   */
+  const setNavigatorSrcBehavior = {
```

Comment:
nit: not abbreviating / changing terminology to keep naming consistent, and updating comment
```suggestion
   * Create a custom behavior to toggle between using Hyperview or externally provided navigator hierarchy
   */
  const setNavigationSourceBehavior = {
```